docs/DeveloperGuide.md

```diff
@@ -341,9 +347,10 @@ Use case ends.
 &nbsp;&nbsp;&nbsp; 1a2. User enters new data  
 &nbsp;&nbsp;&nbsp; 1a3. Use case resumes from step 1
 
-**Use case: Add a lesson**  
```

Comment:
are u sure this renders properly if u remove this line? @luminousleek had an issue where u need to put the space for it to go to the next line